From 41f1a1da57a0cef035d1f78c21bcff3dc3f76f79 Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Tue, 23 Jul 2013 11:26:58 -0400
Subject: [PATCH 2/2] remove systemd-timestamp from sources

No sense in keeping this around if support for reading RD_TIMESTAMP has
been removed.
---
 Makefile.am               |  8 --------
 src/timestamp/timestamp.c | 39 ---------------------------------------
 4 files changed, 49 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 7933de6..d013dfd 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -315,7 +315,6 @@ rootlibexec_PROGRAMS = \
 	systemd-remount-fs \
 	systemd-reply-password \
 	systemd-fsck \
-	systemd-timestamp \
 	systemd-ac-power \
 	systemd-sysctl \
 	systemd-sleep
@@ -1607,13 +1606,6 @@ systemd_fsck_LDADD = \
 	libudev.la
 
 # ------------------------------------------------------------------------------
-systemd_timestamp_SOURCES = \
-	src/timestamp/timestamp.c
-
-systemd_timestamp_LDADD = \
-	libsystemd-shared.la
-
-# ------------------------------------------------------------------------------
 systemd_ac_power_SOURCES = \
 	src/ac-power/ac-power.c
 
